cmd/doctor.go

```diff
@@ -28,10 +32,34 @@ var CmdDoctor = cli.Command{
 	Usage:       "Diagnose problems",
 	Description: "A command to diagnose problems with the current Gitea instance according to the given configuration.",
 	Action:      runDoctor,
+	Flags: []cli.Flag{
+		cli.BoolFlag{
+			Name:  "list",
+			Usage: "List the available checks",
+		},
+		cli.BoolFlag{
+			Name:  "default",
+			Usage: "Run the default checks (if neither run or all is set this is the default behaviour)",
+		},
+		cli.StringSliceFlag{
+			Name:  "run",
+			Usage: "Run the provided checks - if default is set the default checks will also run",
```

Comment:
```suggestion
			Usage: "Run the provided checks - (if --default is set, the default checks will also run)",
```

Parenthesis for consistency.